src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1977,7 +1977,7 @@ namespace RCT2
                     {
                         dst2->SetMazeEntry(src2->GetMazeEntry());
                     }
-                    else if (rideType == RIDE_TYPE_GHOST_TRAIN)
+                    else if (rtd.HasFlag(RIDE_TYPE_FLAG_HAS_LANDSCAPE_DOORS))
```

Comment:
Please revert, see previous comment why. For the S6/T6/legacy .park importers, it’s fine to explicitly check.

Reply:
It was my understanding all of these hardcoded types needed to be gone for RTD objects. Is this not the case?

Reply:
Not for import from SV4/SV6/TD6.